playbooks/roles/heketi-gluster/tasks/storage-creation.yml

```diff
@@ -55,7 +55,8 @@
 # volumtype (replication factor): <VolumeDurability>:number eg. none:1, replicate:2, replicate:3 , disperse:???
 - name: set fact volumetype
   set_fact:
-    volumetype: "{% if (glusterpod_count | int) == 1 %}none:1{% else %}replicate:{{ glusterpod_count }}{% endif %}"
+    volumetype: "{% if (glusterpod_count | int) == 1 %}none:1{% elif (glusterpod_count | int) == 2 %}replicate:2{% else %}replicate:3{% endif %}"
```

Comment:
Can we get this on multiple lines? It is very hard to read.

Reply:
I'll try :-)